packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -238,6 +339,29 @@ export class TriggerManager extends Component {
     })
   }
 
+  cipherToAccount(cipher) {
+    if (!cipher) {
+      return null
+    }
+
+    const customFields = (cipher.fields || []).reduce((fields, field) => {
+      fields[field.name] = field.value
+
+      return fields
+    }, {})
+
+    return {
+      auth: pickBy(
+        {
+          login: cipher.login.username,
+          password: cipher.login.password,
+          ...customFields
+        },
+        value => Boolean(value)
+      )
+    }
+  }
```

Comment:
Should we create a `Cipher` helper / doctype?

Reply:
Or this could be part of the `Account` one? Like `Account::fromCipher`?

Reply:
Deal :)

Reply:
See https://github.com/cozy/cozy-libs/pull/782/commits/712456d95d2e461054896280a139ce713d7403f1